frontend/src/styles/GlobalStyles.ts

```diff
@@ -0,0 +1,70 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyles = createGlobalStyle`
+
+ 
+html {
+    --maxWidth: 1000px;
+    --bs: 0 12px 24px 0 rgba(0,0,0,0,.09)
+    box-sizing: border-box;
+    --strawberry: #f1356d;
+    --lightest: #BFD7ED;
+    --dark: #003B73;
+    --medium: #0074B7;
+    --light: #60A3D9;
```

Comment:
css variables are not declared here. They are declared under the `:root` property like so:

```
:root {
  --strawberry: #f1356d;
    --lightest: #BFD7ED;
    --dark: #003B73;
    --medium: #0074B7;
}
```